client/cypress/end-to-end/batch-comparison-failure-cases.spec.js

```diff
@@ -51,7 +51,6 @@ describe('Batch Comparison', () => {
     cy.findByLabelText('Votes for Candidate/Choice 1').type('2700')
     cy.findByLabelText('Name of Candidate/Choice 2').type('Palpatine')
     cy.findByLabelText('Votes for Candidate/Choice 2').type('2620')
-    cy.findByLabelText('Total Ballots for Contest').type('5320')
```

Comment:
should we be testing for absence of this field?

Reply:
covered that in the unit test i added